assets/app/view/game/game_info.rb

```diff
@@ -151,7 +151,20 @@ def rust_obsolete_schedule
         obsolete_schedule = {}
         @depot.trains.group_by(&:name).each do |_name, trains|
           first = trains.first
+
+          # the first variant in this list should be the base variant
+          # per the train variant initialization.  If other variants
+          # do not have an explicit rust/obsolete, inherit the base
+          # values for display purposes.
+          _, base_variant = first.variants.first
```

Comment:
variants[0]

Reply:
[0] will be a little more complicated because it will need to check that .variants is not empty.  .first correctly sets base_variant to nil if .variants is empty.

Reply:
which doesn't really matter that much because we won't end up using the nil if .variants is empty anyway.

Reply:
I was confused just now because variants is a hash, not an array. I changed to:
```ruby
base_variant = first.variants.values[0]
```